playbooks/common/openshift-node/config.yml

```diff
@@ -7,6 +7,11 @@
   - set_fact:
       openshift_node_debug_level: "{{ t_oo_option_node_debug_level }}"
     when: openshift_node_debug_level is not defined and t_oo_option_node_debug_level != ""
+
+  - name: Fail early if Atomic and old version of Docker
+    include: ../../../roles/openshift_common/tasks/atomic_docker_check.yml
+    when: openshift.common.is_atomic | bool
```

Comment:
To me, just the fact that we need to copy these checks between the master and node playbooks is an indication that we need to do a better job of solving this problem in general.